docs/human_data.md

```diff
@@ -204,3 +204,41 @@ dict_of_tensor = human_data.to()
 gpu0_device = torch.device('cuda:0')
 dict_of_gpu_tensor = human_data.to(gpu0_device)
 ```
+
+## MultiHumanData
+
+MulitHumanData is designed to support multi-human body mesh recovery, who inherits from HumanData. In HumanData, the data can be accessed directly through the index, because the data and the image are in one-to-one correspondence. However, data and image have a many-to-one correspondence in MultiHumanData.
+
+Based on HumanData, MultiHumanData adds a new key named `'optional'` as follows:
+
+```python
+'optional': {
+        'type': dict,
+        'slice_key': 'frame_range',
+        'dim': 0
+    }
+```
```

Comment:
`frame_range` is more like a key and `optional` is more like an attribute.

Reply:
remove `optional` and add a new key named `'frame_range'`
```python
'frame_range': {
        'type': np.ndarray,
        'shape': (-1, 2),
        'dim': 0
}
```